src/index.spec.js

```diff
@@ -37,6 +49,53 @@ describe('Dropzone', () => {
       expect(dropzone.html()).toMatchSnapshot();
     });
 
+    function collectEventListenerCalls(calls) {
+      return calls.reduce((acc, [eventName, ...rest]) => {
+        acc[eventName] = rest; // eslint-disable-line no-param-reassign
+        return acc;
+      }, {});
+    }
+
+    it('prevents stray drops from taking over the browser window by default', () => {
+      const dropzone = mount(<Dropzone><p>Content</p></Dropzone>);
+      expect(dropzone.html()).toMatchSnapshot();
+      expect(document.addEventListener.callCount).toEqual(2);
+      const addCalls = collectEventListenerCalls(document.addEventListener.args);
+
+      const event = { preventDefault: spy() };
```

Comment:
I wondering if this test should be that detailed. I think it will be enough to test that we've called the callback function for appropriate edge cases?